cmd/courier/main.go

```diff
@@ -87,51 +89,61 @@ func main() {
 	}
 
 	// configure our logger
-	logrus.SetOutput(os.Stdout)
-	level, err := logrus.ParseLevel(config.LogLevel)
+	loggerLevel := new(slog.LevelVar)
+	logHandler := slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{Level: loggerLevel})
+	slog.SetDefault(slog.New(logHandler))
+
+	var level slog.Level
+	err := level.UnmarshalText([]byte(config.LogLevel))
 	if err != nil {
-		slog.Error("invalid log level", "level", level)
+		log.Fatalf("invalid log level %s", level)
 		os.Exit(1)
 	}
-	logrus.SetLevel(level)
-
-	// configure golang std structured logging to route to logrus
-	slog.SetDefault(slog.New(utils.NewLogrusHandler(logrus.StandardLogger())))
+	loggerLevel.Set(level)
 
-	log := slog.With("comp", "main")
-	log.Info("starting courier", "version", version)
+	logger := slog.With("comp", "main")
+	logger.Info("starting courier", "version", version)
 
 	// if we have a DSN entry, try to initialize it
 	if config.SentryDSN != "" {
-		hook, err := logrus_sentry.NewSentryHook(config.SentryDSN, []logrus.Level{logrus.PanicLevel, logrus.FatalLevel, logrus.ErrorLevel})
-		hook.Timeout = 0
-		hook.StacktraceConfiguration.Enable = true
-		hook.StacktraceConfiguration.Skip = 4
-		hook.StacktraceConfiguration.Context = 5
+		err := sentry.Init(sentry.ClientOptions{
+			Dsn:           config.SentryDSN,
+			EnableTracing: false,
+		})
 		if err != nil {
-			log.Error("unable to configure sentry hook", "dsn", config.SentryDSN, "error", err)
+			log.Fatalf("error initiating sentry client, error %s, dsn %s", err, config.SentryDSN)
 			os.Exit(1)
 		}
-		logrus.StandardLogger().Hooks.Add(hook)
+
+		defer sentry.Flush(2 * time.Second)
+
+		logger = slog.New(
+			slogmulti.Fanout(
```

Comment:
We need to use multi handlers here so that we can log the error both to sentry and the text logs on the servers